views/settings/accountstatus.tt.text

```diff
@@ -1,6 +1,6 @@
 ;; -*- coding: utf-8 -*-
-.btn.status|notes=Submit button for form
-.btn.status=Change Status
+.btn.change|notes=Submit button for form
```

Comment:
whoa, I had no idea this `|notes` thing was a thing.

Reply:
I haven't seen it used very often! I never think to use it either.